sonic-xcvrd/xcvrd/pm_mgr.py

```diff
@@ -0,0 +1,497 @@
+                return
+            
+            #update the port list
+            if lport not in self.pm_port_list:
+                self.pm_port_list.append(lport)
+            else:
+                self.log_notice("Port number:{} already present in port list :: Spurious request\n".format(lport))
+                return
+            pm_win_dict = {}
+            # Set the TRANSCEIVER_PM_WINDOW_STATS keys for the port with default value
+            for key in ["window{}".format(win_num) for win_num in range(WINDOW_60SEC_START_NUM, WINDOW_24HRS_END_NUM + 1)]:
+                pm_win_dict[key] = "N/A" 
+        
+            #Update the table in DB    
+            state_pm_win_stats_tbl = self.xcvr_table_helper.get_pm_window_stats_tbl(asic_id)
+            fvs = swsscommon.FieldValuePairs([(k, v) for k, v in pm_win_dict.items()])
+            state_pm_win_stats_tbl.set(lport, fvs)
+            self.log_notice("Port add event:{} added to pm port list".format(port_change_event.port_name))
+        elif port_change_event.event_type == port_change_event.PORT_DEL and port_change_event.table_name == 'TRANSCEIVER_INFO':
+
+            if self.platform_chassis.get_sfp(physical_port).get_presence():
+                self.log_error("TRANSCIEVER_INFO table for the port {} is deleted but transciever is present".format(lport))
+                return
+            #Remove the port from port list and delete the TRANSCEIVER_PM_WINDOW_STATS table for the port 
+            self.pm_port_list.remove(lport)
+            state_pm_win_stats_tbl = self.xcvr_table_helper.get_pm_window_stats_tbl(asic_id)
+            if state_pm_win_stats_tbl:
+                state_pm_win_stats_tbl._del(lport)
+                self.log_notice("Port delete event:{} deleted from pm port list".format(port_change_event.port_name))
+
+    # Update the PM statistics in string format to respective window key to 
+    # TRANSCEIVER_PM_WINDOW_STATS_TABLE in state DB.
+    def set_pm_stats_in_pm_win_stats_tbl(self, asic, lport, window, pm_data_str):
+        state_pm_win_stats_tbl = self.xcvr_table_helper.get_pm_window_stats_tbl(asic)
+        fvs = swsscommon.FieldValuePairs([(window, str(pm_data_str))])
+        state_pm_win_stats_tbl.set(lport, fvs)
+
+    # Retrieve the PM statistics from respective window key from.
+    # TRANSCEIVER_PM_WINDOW_STATS_TABLE - state DB.
+    def get_pm_stats_in_pm_win_stats_tbl(self, asic, lport, window):
+        state_pm_win_stats_tbl = self.xcvr_table_helper.get_pm_window_stats_tbl(asic)
+        status, fvs = state_pm_win_stats_tbl.get(str(lport))
+        stats_tbl_tuple = dict((k, v) for k, v in fvs)
+        if stats_tbl_tuple != "":
+            return (stats_tbl_tuple.get(window, ""))
+        else:
+            return None
+
+    # 
+    # input args: asic index, logical port, start and end window number of a PM window granularity, 
```

Comment:
could we have standard python docstring just like below function?
https://github.com/sonic-net/sonic-platform-daemons/blob/c1c43f686ce29c005de8e7e87085c440bc7af998/sonic-xcvrd/xcvrd/xcvrd.py#L1112-L1126

Reply:
Corrected the the format/comment